rmf_demos.repos

```diff
@@ -19,3 +19,7 @@ repositories:
     type: git
     url: https://github.com/osrf/rmf_demos.git
     version: master
+  thirdparty/menge_core:
+    type: git
+    url: https://github.com/osrf/menge_core.git
+    version: master
```

Comment:
It would be better to pin the version to a specific tag or commit to avoid surprises.

Reply:
I don't think it would be changed much, I am open to create a tag but I would probably wait until we release it, I guess we could need a fair bit of changes to package it and I don't want to risk forgetting to update the repos here

Reply:
OK, I can live with that. We should make sure that each release branch of `rmf_demos` points to the specific commit of `menge` that it works with, though. Otherwise we may find old releases suddenly stop compiling or running.

Reply:
Sounds good, we will create the tag in the first release that includes the menge_core package :heavy_check_mark: